static/styles/zulip.scss

```diff
@@ -811,6 +811,23 @@ td.pointer {
     }
 }
 
+.topic_chevron_color_bar {
+    margin-left: 1px;
+    height: 100%;
+    display: inline-block;
+    position: absolute;
+
+    .topic_chevron_icon {
+        margin-top: 6px;
+        margin-left: 10px;
+        font-size: 11px;
+    }
+}
+
+.recipient_bar_right_side_spacing {
+    margin-right: 14px;
+}
```

Comment:
Added this as having the color bar match up to with the recipient bar required having only the left side of a `box-shadow (otherwise the widths were impossible to match up). In order to fill the recipient bar's height we needed `position: absolute` in `.topic_chevron_color_bar` which caused margin's of the `topic_chevron` parent-child elements to not work as expected.

So I added this div in order to space out the topic name from the icon.  Not sure if this is the best way.